### DISCUSSION TITLE
Unsolvable captcha

### DISCUSSION THREAD

**Discussion by jovanjanevski** [community/community#114532] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I have solved 15 challenges of your captcha and it still won't let me create an organisation.
What kind of disability do i have?

**Comment by ghostinhershell** [comment]
Hi @jovanjanevski, 

Thanks so much for reaching out in our community! 👋🏾 

> I have solved 15 challenges of your captcha and it still won't let me create an organisation.

To clarify, do you see any error messages when you do complete the CAPTCHA? For example, does it show that the CAPTCHA is incorrect when you complete it? Or are you still unable to create an organization even though you have successfully completed the CAPTCHA? Any additional information that you share should help us help you.  

> What kind of disability do i have?

I don't think anyone here in the Community has the ability to diagnose a disability but I do encourage discussing any concerns that you have with your primary care doctor.

Thanks!
